packages/server-core/src/media/file-browser/file-browser.class.ts

```diff
@@ -152,7 +151,7 @@ export class FileBrowserService
     }
 
     return {
-      total,
+      total: result.length,
```

Comment:
this will take the sliced result, what is there currently is correct

Reply:
I did this for the user seeing higher page count than the total items when going to the root directory in file browser. As the total returns all projects count while the result only has the projects, user has access to.

Reply:
reverted

Reply:
I think this needs to be restored to what it was here, since that was the intended behavior.

Reply:
@MoizAdnan

Reply:
This was an older change and I had reverted it sometime back. As this PR https://github.com/EtherealEngine/etherealengine/pull/9836 ,  fixes the total bug